sticker-utils/src/bin/sticker-train.rs

```diff
@@ -5,29 +5,80 @@ use std::io::BufReader;
 use clap::Arg;
 use failure::Error;
 use indicatif::ProgressStyle;
+use ordered_float::NotNan;
 use stdinout::OrExit;
 use sticker::depparse::{RelativePOSEncoder, RelativePositionEncoder};
 use sticker::tensorflow::{
-    ConllxDataSet, DataSet, LearningRateSchedule, TaggerGraph, TaggerTrainer,
+    ConllxDataSet, DataSet, LearningRateSchedule, PlateauLearningRate, TaggerGraph, TaggerTrainer,
 };
 use sticker::{CategoricalEncoder, LayerEncoder, Numberer, SentVectorizer, SentenceEncoder};
 use sticker_utils::{
     sticker_app, CborRead, Config, EncoderType, LabelerType, ReadProgress, TomlRead,
 };
 
 static CONFIG: &str = "CONFIG";
+static INITIAL_LR: &str = "INITIAL_LR";
+static LR_SCALE: &str = "LR_SCALE";
+static LR_PATIENCE: &str = "LR_PATIENCE";
+static PATIENCE: &str = "PATIENCE";
 static TRAIN_DATA: &str = "TRAIN_DATA";
 static VALIDATION_DATA: &str = "VALIDATION_DATA";
 
+pub struct LrSchedule {
+    pub initial_lr: NotNan<f32>,
+    pub lr_scale: NotNan<f32>,
+    pub lr_patience: usize,
+}
+
 pub struct TrainApp {
     config: String,
+    lr_schedule: LrSchedule,
+    patience: usize,
     train_data: String,
     validation_data: String,
 }
 
+impl TrainApp {
+    pub fn lr_schedule(&self) -> PlateauLearningRate {
+        PlateauLearningRate::new(
+            self.lr_schedule.initial_lr.into_inner(),
+            self.lr_schedule.lr_scale.into_inner(),
+            self.lr_schedule.lr_patience,
+        )
+    }
+}
+
 impl TrainApp {
     fn new() -> Self {
         let matches = sticker_app("sticker-train")
+            .arg(
+                Arg::with_name(INITIAL_LR)
+                    .long("lr")
+                    .value_name("LR")
+                    .help("Initial learning rate")
+                    .default_value("0.01"),
```

Comment:
Iirc, 0.01 is a fairly high value, are you using that number in some models?

Reply:
It's my default for RNN models.

Reply:
I see, I was probably thinking about the clause models. I think i used 0.005 or 0.001 on those.

Reply:
The convolution models typically even diverge with 0.01, but I guess RNNs are currently the best default option. It's definitely something we should add to the documentation.